index.js

```diff
@@ -22,6 +22,10 @@ async function run(options, plugins) {
   if (!isCi && !options.dryRun && !options.noCi) {
     logger.log('This run was not triggered in a known CI environment, running in dry-run mode.');
     options.dryRun = true;
+  } else {
+    // When running on CI, prevent the `git` CLI to prompt for username/password
```

Comment:
I would suggest we add the link to this PR here for easier reference in future: https://github.com/semantic-release/semantic-release/issues/699#issuecomment-373445979

Reply:
Should the comment and [GitHub git blame](https://help.github.com/articles/tracing-changes-in-a-file/) be enough?
Personally I use Git blame really often, but not sure if it's a well known feature.

I'm not sure if in general it's better to reference PR/issues as comment versus using GitHub/Git feature. Or at least I don't know in which cases it worst it to reference an PR/issue in a comment. Any suggestion?